Classes/common/CDTIncrementalStore.h

```diff
@@ -0,0 +1,121 @@
+//
+//  CDTIncrementalStore.h
+//
+//
+//  Created by Jimi Xenidis on 11/18/14.
+//
+//
+
+#import <CoreData/CoreData.h>
+#import <CloudantSync.h>
+
+extern NSString *const kCDTISErrorDomain;
+extern NSString *const kCDTISException;
```

Comment:
I recommend dropping the 'k' prefix on these constant names.  Neither Apple nor Cloudant uses the 'k' convention (Cloudant's error domain is CDTDatastoreErrorDomain).